spec/jobs/update_plage_ouvertures_expirations_job_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe UpdatePlageOuverturesExpirationsJob, type: :job do
+  subject do
+    UpdatePlageOuverturesExpirationsJob.perform_now
+  end
+
+  context "when plage ouverture are expired" do
+    let!(:plage_ouverture_reguliere) { create(:plage_ouverture, recurrence: Montrose.every(:week, until: Time.now).to_json) }
```

Comment:
why `to_json` ?

Reply:
if you're not using the created AR records you can also simply create them in a before block